salesforce-composite/2.7.0/modules/ROOT/pages/salesforce-composite-connector-reference.adoc

```diff
@@ -0,0 +1,1366 @@
+
+Use the authorization server's URL for the audience value: `+https://login.salesforce.com+`, `+https://test.salesforce.com+`. If it is a community implementation, use `+https://community.force.com/customers+`. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|======================
+
+[[composite-config_oauth-saml]]
+=== OAuth SAML
+
+
+==== Parameters
+[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+|======================
+| Name | Type | Description | Default Value | Required
+| TLS configuration a| <<tls-configuration>> |  Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, then, at a minimum, the user must configure the keystore in the `tls:context` child element of the `listener-config`.  |  | 
+| Host a| String |  Hostname of the proxy. Required if a proxy is used. If this property is not set, then no proxy is used. |  | 
+| Port a| Number |  Port of the proxy. If the Host is set, then this property is required and the value cannot be a negative number. |  | 
+| Username a| String |  Username used to authenticate against the proxy. If this property is not set then no authentication is used against the proxy. |  | 
+| Password a| String |  Password used to authenticate against the proxy. |  | 
+| Response timeout a| Number |  Wait time until a response is received. This attribute works in tandem with Response Timeout Unit. Values less than one millisecond are converted to `0`. |  5 | 
+| Response timeout unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use for the Response Timeout parameter. |  SECONDS | 
+| Connection Timeout a| Number |  The socket connection timeout value. This attribute works in tandem with Connection Timeout Unit. |  | 
+| Connection Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Connection Timeout. |  SECONDS | 
+| Consumer Key a| String |  Consumer key for the Salesforce connected app. |  | *x*
+| Key Store a| String |  Path to the keystore used to sign data during authentication. |  | *x*
+| Store Password a| String |  Password of keystore. |  | *x*
+| Certificate Alias a| String |  Alias of the certificate. |  | 
+| Principal a| String |  Username of the Salesforce user on whose behalf to take action. |  | *x*
+| Token Endpoint a| String | URL pointing to the server responsible for providing the authentication token. The default is `+https://login.salesforce.com/services/oauth2/token+`. 
```

Comment:
Same comment as before about the default. I won't mark these again. It will make that column wide, so it may be best not to change these.